src/cliques/cliquer/CMakeLists.txt

```diff
@@ -12,18 +12,17 @@ target_include_directories(
   cliquer
   PRIVATE
   ${CMAKE_SOURCE_DIR}/include
+  ${CMAKE_BINARY_DIR}/include
   ${CMAKE_BINARY_DIR}/src
 )
 
 if (BUILD_SHARED_LIBS)
   set_property(TARGET cliquer PROPERTY POSITION_INDEPENDENT_CODE ON)
 endif()
 
-if (WIN32)
-  # Since these are included as object files, they should call the
-  # function as is (without declspec)
-  target_compile_definitions(cliquer PRIVATE IGRAPH_STATIC)
-endif()
+# Since these are included as object files, they should call the
+# function as is (without declspec)
```

Comment:
Now that `IGRAPH_STATIC` or `igraph_EXPORTS` are also valid on other platforms besides Windows, should we perhaps clarify this comment a bit? (The same of course holds for the other locations).

Reply:
Done (see last commit)